CHANGELOG.md

```diff
@@ -1,29 +1,29 @@
 ## 3.0.3
-- Use a less error-prone way of specifying gem files (#662)
-- Update rake requirement from `>= 10.4, < 13.0` to `>= 10.4, < 14.0` (#659)
-- Bump nokogiri from 1.6.6.2 to 1.10.4 in /spec/integration/rails_4.2.0 (#655)
-- Default annotate models to true in config generated by `rails g annotate:install` (#671)
-- Bump loofah from 2.3.0 to 2.3.1 in /spec/integration/rails_4.2.0 (#681)
+- Use a less error-prone way of specifying gem files (https://github.com/ctran/annotate_models/pull/662)
```

Comment:
Would it be possible to have links in the format [#662](https://github.com/ctran/annotate_models/pull/662)?

`[#xxx](https://github.com/ctran/annotate_models/pull/xxx)`

This would make the changelog links shorter but also be able to use the link.

Reply:
Thank you for seeing.
Fixed.